### DISCUSSION TITLE
It isnt working

### DISCUSSION THREAD

**Discussion by Mackb971** [Prodigy-Hacking/ProdigyMathGameHacking#833] [(A) Anything]
When I go on to prodigy this comes up Oh No! Something went wrong while trying to connect to the server! Try reloading this page. If this error continues to appear, join our Discord for support, or create an issue on GitHub. More info TypeError: Failed to fetch and when I press ok it sends me to discord so could any of you guys help me get it working

**Comment by Mackb971** [comment]
pls help me

**Reply by lukeschwinn001** [reply]
B̴̟͎̻͌̓́z̴̻͚̈́͠͝z̸̙͙͕̈́̿͆T̴̝̘̞̽͊͝ - ApparenB̴̟͎̻͌̓́z̴̻͚̈́͠͝z̸̙͙͕̈́̿͆T̴̝̘̞̽͊͝ tly dDiscord B̴̟͎̻͌̓́z̴̻͚̈́͠͝z̸̙͙͕̈́̿͆T̴̝̘̞̽͊͝ was compromised so I B̴̟͎̻͌̓́z̴̻͚̈́͠͝z̸̙͙͕̈́̿͆T̴̝̘̞̽͊͝ can't even go B̴̟͎̻͌̓́z̴̻͚̈́͠͝z̸̙͙͕̈́̿͆T̴̝̘̞̽͊͝ there

**Reply by PatheticMustan** [reply]
It means your school blocked the hacks... If you don't have a web filter, it might be an actual issue

**Reply by NessiePUYOY** [reply]
same here i am able to download the hack but not able to use them at all. It says the same thing as well please update it to be in use for everyone again because schools are really strict about hacks and coding and it wont be unblocked so it would be nice if you can update the hack to work again please :)

**Reply by LeoBadeaux** [reply]
> same here i am able to download the hack but not able to use them at all. It says the same thing as well please update it to be in use for everyone again because schools are really strict about hacks and coding and it wont be unblocked so it would be nice if you can update the hack to work again please :)

We can't just magically _unblock_ it from our end.

**Reply by FlinxDa1** [reply]
This is happening to me and I am using a personal computer. Are the hacks down?

**Reply by FlinxDa1** [reply]
@PatheticMustan

**Reply by NessiePUYOY** [reply]
I Guess i can never use this hacking thing again :/

**Reply by SakuraKitsunee** [reply]
wait @PatheticMustan if they have the extension but can't use it, can they use the ItemStacker.js in the hacks folder with the bookmarklet maker to get all the items still, or am I wrong

**Reply by ghost** [reply]
bookmarklet dosent work for meh pls help
